local_settings.example.py

```diff
@@ -257,6 +265,36 @@
 
 # }}}
 
+# {{{ overriding built-in templates and staticfiles
+# Uncomment the following to enable templates overriding. It should be configured
+# as a list/tuple of path(s).
+# For example, if you the templates are in a folder named "my_templates" in the
+# root dir of the project, with base.html (project template), course_base.html,
+# and sign-in-email.txt (app templates) etc., are the templates you want to
+# override, the structure of the files should look like:
+#    ...
+#    relate/
+#    local_settings.py
+#    my_templates/
+#        base.html
+#        ...
+#        templates/
+#            course/
+#                course_base.html
+#                sign-in-email.txt
+#                ...
+#
+
+# import os.path
+# MY_TEMPLATES_DIRS = [os.path.join(os.path.dirname(__file__), "my_templates"),
+#                      os.path.join(os.path.dirname(__file__), "my_other_templates")]
+
+# Uncomment the following to enable staticfile overriding, It should be configured
+# as a list/tuple of path(s)
+
+# MY_STATICFILES_DIRS = [os.path.join(os.path.dirname(__file__), "staticfiles"),]
+
+# }}}
 
 # {{{ docker
```

Comment:
Not a fan of `MY_` prefixes--I'd prefer if these said `RELATE_` instead.

Reply:
Updated to `RELATE_OVERRIDE_`